sound/soc/sof/sof-of-dev.c

```diff
@@ -75,6 +110,12 @@ static int sof_of_probe(struct platform_device *pdev)
 	sof_pdata->fw_filename_prefix = sof_pdata->desc->default_fw_path;
 	sof_pdata->tplg_filename_prefix = sof_pdata->desc->default_tplg_path;
 
+	ret = sof_of_parse(pdev);
+	if (ret < 0) {
+		dev_err(dev, "error: failed to parse DSP DT node\n");
```

Comment:
can you also print ret in debug message ?

Reply:
@RanderWang good point.

Reply:
@RanderWang actually i don't think is needed. Probe when fails it will return the error stored in ret.